### PR TITLE
Remove unused wallet migration

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,52 +6,20 @@ import './shim.js';
...
 }
 
-class BlueAppComponent extends React.Component {
-  state = {
-    isMigratingData: true,
-  };
-
-  componentDidMount() {
-    const walletMigrate = new WalletMigrate(this.setIsMigratingData);
-
```
Comment (Kamill90) [inline index.js:26]: don't we need it?

File: index.js [hunk 2]
```diff
@@ -6,52 +6,20 @@ import './shim.js';
...
-    walletMigrate.start();
-  }
-
-  setIsMigratingData = async () => {
-    SplashScreen.hide();
-    this.setState({
```
Comment (Kamill90) [inline index.js:32]: how do you hide the splashscreen then?

File: index.js (hunk 1 above)
Comment (KedziaPawel) [reply index.js:26]: seems to be useless, I think it was a migration that BlueWallet needed. Do you know for what we need it?

File: index.js (hunk 2 above)
Comment (KedziaPawel) [reply index.js:32]: the application seems to work fine without it

Comment (Kamill90) [reply index.js:32]: clean build and you should see eternal splashScreen. SplashScreen.hide() needs to be called in the root component.
https://github.com/crazycodeboy/react-native-splash-screen#usage

Comment (KedziaPawel) [reply index.js:32]: added

**Review by Kamill90 — APPROVED** [review]

**Pull request merged**